vllm/engine/arg_utils.py

```diff
@@ -1341,9 +1341,10 @@ def _is_v1_supported_oracle(self, model_config: ModelConfig) -> bool:
             return False
 
         if (self.pipeline_parallel_size > 1
-                and self.distributed_executor_backend not in ["ray", "mp"]):
+                and self.distributed_executor_backend
+                not in ["ray", "mp", "external_launcher"]):
```

Comment:
nit: Use immutable tuple.

```suggestion
                not in ("ray", "mp", "external_launcher")):
```